.travis.yml

```diff
@@ -63,14 +54,6 @@ install:
     - echo "install"
     - conda env create --file ci/requirements-$CONDA_ENV.yml
     - source activate test_env # all envs are named test_env in the yml files
-    # needed to make sure that pandas is compiled against the right
-    # version of numpy
-    - if [[ "$CONDA_ENV" == "py27-min" ]]; then
```

Comment:
@wholmgren Do we need to define a new minimum environment in Python 3?

Reply:
Yes, good catch. I'm pretty sure you'll be able to pin the minimum version numbers in the pip section of the file. I'm less sure about the conda section of the file. Either way is fine with me. I don't think we'll need anything special after that.

Reply:
Well, I was wrong -- I'm not able to easily create a python 3.5 environment with the current numpy/pandas minimums on my mac. I suspect it's possible but I don't think it's worth the effort to struggle against theses old packages. I can easily build and test the environment with pandas 0.17:

```
# requirements-py35-min.yml
name: test_env
channels:
    - defaults
dependencies:
    - python=3.5
    - pip
    - pip:
        - numpy==1.10.1
        - pandas==0.17.0
        - pytz
        - nose
        - pytest
        - pytest-mock
        - pytest-timeout
        - coveralls
```

We should also specify a newer version of conda in the travis config. Latest is fine.

Only catch is that `test_sun_rise_set_transit_spa` claims it needs pandas 0.17, but it apparently needs 0.18.